front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.tsx

```diff
@@ -72,11 +65,17 @@ const ProjectMoreActionsMenu = ({
       actions.push({
         handler: async () => {
           setLoadingState('copying', true);
-          await handleCallbackError(
-            () => copyProject(projectId),
-            formatMessage(messages.copyProjectError)
-          );
-          setLoadingState('copying', false);
+
+          copyProject(projectId, {
+            onSuccess: () => {
+              setError(null);
+              setLoadingState('deleting', false);
```

Comment:
I wonder if we shouldn't handle this loading and erros state with React query directly?

Reply:
I guess loading is pretty straightforward, but I'm not sure for error handling... for `react-query` to handle it properly it must be in the `CLErrors` format, right? But here if it's setting a general error message my guess is that the response is not in the `CLError` format. Or can `react-query` also deal with other types of errors?

Reply:
Yes, perhaps we should leave it as is for now. We are planning to do some work on error handling in the future where we can address these cases.